### ISSUE TITLE
Deep link for actions

### ISSUE THREAD

**Issue by danshearer** [pronovic/hcoop-meetbot#14]
Thanks for the meetbot. We're using it in LumoSQL as per our [Meetbot HOWTO](https://lumosql.org/src/lumosql/doc/tip/doc/lumosql-meetbot.md).

The most important outcome of a meeting is usually the action items. How about each action item have a unique link so that it is possible to point to individual action items forever after? 

For example we might say in conversation:

```
     I had a thought about Sally's action  
   [from the last meeting](https://example.org/meetings/meeting-2022-01-02.html#sally-action-item1)
    and I think we can make it easier....
```

Thanks,

Dan

**Comment by pronovic** [comment]
Hi @danshearer , glad that you're finding the meetbot helpful.  I'm open to this enhancement.  I won't have time to be in the code here for a week or two, but I've got it on my list.  I'll have to think it through a little, so I may come back with other questions later.  I'll also accept a PR if you already have an implementation in mind.

**Comment by danshearer** [comment]
@pronovic said:

>  ... accept a PR if you already have an implementation in mind.

I had a look at that, but I agree with your words in DEVELOPER.md:

> Limnoria (Supybot) plugins are quite specialized, with a standard code structure and a dedicated test framework.

My first thought was that it might be possible just by editing the last stanza in hcoopmeetbotlogic/templates/minutes.html , but then I realised there are no links at all in the body of the meeting notes so it will need a change in writer.py to introduce this new concept (which has several other potential uses I guess.) At which point I raised an Issue instead of trying to create a PR :-)

**Comment by pronovic** [comment]
@danshearer I released [v0.5.0](https://pypi.org/project/hcoop-meetbot/0.5.0/) with a first pass at this enhancement.  Take a look and see if it meets your needs.  If not, re-open this issue and we can work on a better solution.
